src/service/configService.ts

```diff
@@ -36,8 +36,8 @@ const configService = {
         'AUTH_USERNAME' : 'admin',
         'AUTH_PASSWORD' : '5f4dcc3b5aa765d61d8327deb882cf99',
         'FALLBACK_FILENAME_SUFFIX' : 'WEB.H264-BBC',
-        'MOVIE_FILENAME_TEMPLATE' : '{{#if synonym}}{{synonym}}{{else}}{{title}}{{/if}}.BBC.WEB-DL.AAC.2.0.{{quality}}.H.264',
-        'TV_FILENAME_TEMPLATE' : '{{#if synonym}}{{synonym}}{{else}}{{title}}{{/if}}.S{{season}}E{{episode}}.WEB.{{quality}}.H.264.BBC',
+        'MOVIE_FILENAME_TEMPLATE' : '{{#if synonym}}{{synonym}}{{else}}{{title}}{{/if}}.WEBDL.{{quality}}-BBC',
```

Comment:
Tested this format in Sonarr's parse tool and it correctly picks up the release group as BBC with this format, but happy to change it back.